### PR TITLE
Added Synthetic and Inactive blood

### PR DESCRIPTION
**Pull request by CodedCrow**
## About the PR
Added synthetic blood, and inactive blood

Inactive blood: While it does count as food towards arachnee it also drains them of some asphyxiation to discourage them from doing so. 
The recipe is 10 carbon, 10 hydrogen, 10 iron, 1 plasma for 25 inactive blood. 
Because its a chemical it can be turned into pills and added to foods.

Synthetic blood: Similar to blood in most aspects but if digested by an arachnee it only creates half the Omnizine.
The recipe is 1 Inactive blood and 1 oxygen for 1 Synthetic blood.
It can also be turned into food and pills, this allows arachnee to have a large supply to food on their person and thus a large supply of Omnizine. 

**Media**
Crafting the recipe:
https://github.com/DeltaV-Station/Delta-v/assets/114866135/e40fc7a4-e2a0-498c-ba78-8a341b4c2465

And here's a short video of me after ingesting it.
https://github.com/DeltaV-Station/Delta-v/assets/114866135/aaa4d136-bbef-4dca-b762-b3c8bba06d95

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: CodedCrow
- add: Added Inactive blood and its crafting recipe
- add: Added Synthetic blood and its crafting recipe

### CODE REVIEW

File: Resources/Locale/en-US/reagents/meta/biological.ftl [hunk 1]
```diff
@@ -6,3 +6,9 @@ reagent-desc-slime = You thought this was gradient blood at first, but you were
 
 reagent-name-ichor = ichor
 reagent-desc-ichor = An extremely potent regenerative chemical, perfected by space fauna evolution. Produced in the dragon's digestive system, it is seen as an exotic commodity due to the gargantuan effort of hunting for it.
+
+reagent-name-syntheticblood = synthetic blood
+reagent-desc-syntheticblood = Synthetic blood, a cruetly free alternative for Arachnees. Originally Developped by Lia Rogers.
+
+reagent-name-inactiveblood = inactive blood
+reagent-desc-inactiveblood = Synthetic blood without the oxygen, causes asphyxiation if drank by an Arachnee
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/reagents/meta/biological.ftl]:
```suggestion
reagent-desc-inactiveblood = Synthetic blood without the oxygen, causes asphyxiation if drank by an Arachne
```

File: Resources/Locale/en-US/reagents/meta/biological.ftl [hunk 2]
```diff
@@ -6,3 +6,9 @@ reagent-desc-slime = You thought this was gradient blood at first, but you were
 
 reagent-name-ichor = ichor
 reagent-desc-ichor = An extremely potent regenerative chemical, perfected by space fauna evolution. Produced in the dragon's digestive system, it is seen as an exotic commodity due to the gargantuan effort of hunting for it.
+
+reagent-name-syntheticblood = synthetic blood
+reagent-desc-syntheticblood = Synthetic blood, a cruetly free alternative for Arachnees. Originally Developped by Lia Rogers.
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/reagents/meta/biological.ftl]:
```suggestion
reagent-desc-syntheticblood = Synthetic blood, a cruelty free alternative for Arachne. Originally Developed by Lia Rogers.
```

File: Resources/Prototypes/Recipes/Reactions/chemicals.yml [hunk 3]
```diff
@@ -370,3 +370,27 @@
...
+    Hydrogen:
+      amount: 10
+    Carbon:
+      amount: 10
+    Iron:
+      amount: 10
+    Plasma:
+      amount: 1
+  products:
+    InactiveBlood: 25
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Recipes/Reactions/chemicals.yml]:
```suggestion
    Hydrogen:
      amount: 1
    Carbon:
      amount: 1
    Iron:
      amount: 1
    Plasma:
      amount: 0.1
  products:
    InactiveBlood: 2.5
```

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Recipes/Reactions/chemicals.yml]: I think the plasma would be better as a catalyst too

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Reactions/chemicals.yml (hunk 3 above)
Comment (CodedCrow) [reply Resources/Prototypes/Recipes/Reactions/chemicals.yml]: We discussed it, and the reasoning behind the choice was to give cargo more work to do. Blood also has plasma in it; not the same kind of plasma, but plasma nontheless. It was also added to make this less overall powerful: we still want Arachne to have to find their food, not just mass produce it in a chem lab.

**Review by DEATHB4DEFEAT — DISMISSED** [review]

File: Resources/Prototypes/Reagents/biological.yml [hunk 4]
```diff
@@ -96,6 +96,77 @@
...
+        damage:
+          types:
+            Asphyxiation: 0.6
+        conditions:
+        - !type:OrganType
+          type: Vampiric
+        reagent: Water
+        amount: 0.15
+
   id: Fat
```
Comment (DebugOk) [inline Resources/Prototypes/Reagents/biological.yml:174]: Linter is failing because you removed the `- type: reagent` here

**Review by CodedCrow — COMMENTED** [review]
Thank you for the help

**Review by DebugOk — APPROVED** [review]
Looks good enough

**Pull request merged**